File: C++/MaxMin.cpp
#include<iostream>
#include<vector>
#include<limits.h>
using namespace std;

int main(){
    int n;
    cin>>n;
    vector<int> arr(n,0);

    for(int i=0;i<n;i++){
        cin>>arr[i];
    }

    int maximum=INT_MIN;
    int minimum=INT_MAX;

    for(int i=0;i<n;i++){
        maximum=max(maximum,arr[i]);
        minimum=min(minimum,arr[i]);
    }

    cout<<maximum<<" "<<minimum<<endl;
}

File: Java/MaxMin.java
import java.util.Scanner;

public class MaxMin {

    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);

        int n=sc.nextInt();
        int []arr=new int[n];

        for(int i=0;i<n;i++){
            arr[i]=sc.nextInt();
        }

        MaxMin(arr,n);

    }

    private static void MaxMin(int[] arr, int n) {

        int max=Integer.MIN_VALUE;
        int min=Integer.MAX_VALUE;

        for(int i=0;i<n;i++){
            max=Math.max(max,arr[i]);
            min=Math.min(min,arr[i]);
        }

        System.out.println(max+" "+min);
    }
}
